docs/_rtd/data_types.md

```diff
@@ -90,3 +91,70 @@ The CSV should contain the following columns
 * `fov`: name of the FOV the cell comes from
 * `label`: the name of the segmentation label
 * A set of expression columns defining the properties of each cell desired for clustering
+
+---
+
+Name: AnnData   
+Type: anndata.AnnData   
+Created by: [ConvertToAnnData](https://ark-analysis.readthedocs.io/en/latest/_markdown/ark.utils.html#ark.utils.data_utils.ConvertToAnnData)    
+Used by: [anndata_conversion.ipynb](https://github.com/angelolab/ark-analysis/blob/main/templates/anndata_conversion.ipynb)
+
+<p align="center">
+  <img width="50%" src="../_images/anndata_schema.png" alt="AnnData Schema"/>
+</p>
+
+
+`AnnData` is a data structure consisting of matrices, annotated by DataFrames and Indexes.
+
+A `AnnData` object is composed of the following components:
+
+- **X**
+- **var**
+- **obs**
+- **obsm**
+- **obsp**
+- **varm**
+- **varp**
+
+Each of these components have specific use cases and will be described below:
+
+#### 1. X, var, obs
+
+<p align="center">
+  <img width="50%" alt="image" src="https://github.com/angelolab/ark-analysis/assets/8909315/a5011077-d350-4aab-b8f8-609b11087bba">
+</p>
+
+- `X` is a matrix of shape `(n_obs, n_vars)` where `n_obs` is the number of observations and `n_vars` is the number of variables. 
```

Comment:
Even though this data structure is quite flexible, to start with we're going to be using it primarily to store the cell table. I think having specific examples for each matrix, like you do the latter half of the matrices, will be helpful when people are browsing. We can always update this in the future once we have multiple types of AnnData objects, but for now filling it in assuming it will be a cell table I think will make it easier to understand.